pkg/controller/logstorage/initializer/initializing_controller.go

```diff
@@ -42,9 +42,15 @@ import (
 var log = logf.Log.WithName("controller_logstorage")
 
 const (
-	DefaultElasticsearchStorageClass = "tigera-elasticsearch"
-	TigeraStatusName                 = "log-storage"
-	defaultEckOperatorMemorySetting  = "512Mi"
+	DefaultElasticsearchStorageClass     = "tigera-elasticsearch"
+	TigeraStatusName                     = "log-storage"
+	TigeraStatusLogStorageAccess         = "log-storage-access"
```

Comment:
These look like they are duplicating the constants from the sub-controllers. I think we want to define a single constant for each so that they don't get out-of-sync.

Either these should each be defined in the sub-packages where they are used, or we should remove the constants from those sub-packages as part of this PR and instead use the values from here.